### ISSUE TITLE
NAU8822 driver ?

### ISSUE THREAD

**Issue by ad6fp** [XGudron/UA3REO-DDC-Transceiver#50]
I noticed in the code base a driver for a NAU8822 codec but no use of this codec in the schematics for any Wolf variant.
Will this be replacing the WM8731 codec which is no longer an active part and it's corresponding AF amplifier ?

**Comment by XGudron** [comment]
No, no replacement is expected. The NAU8822 codec is used in the Wolf-Mini subproject

**Comment by ad6fp** [comment]
Thank you.


On Dec 12, 2023, at 10:10 PM, Dmitry Panin ***@***.***> wrote:



No, no replacement is expected. The NAU8822 codec is used in the Wolf-Mini subproject
